src/lib.rs

```diff
@@ -27,324 +27,44 @@ extern crate glutin;
 
 #[macro_use]
 mod macros;
-mod camera;
+pub mod camera;
+pub mod controls;
```

Comment:
This plural is OK, because `controls::Orbit` makes more sense than `control::Orbit`.